plugins/inputs/prometheus/sample.conf

```
@@ -66,6 +66,11 @@
   ## Default is 60 seconds.
   # pod_scrape_interval = 60
 
+  ## Content length limit
+  ## When set, telegraf will drop responses with length larger than the
+  ## configured value in bytes.
+  # content_length_limit = 0
```

With the change to `config.Size` I would set
```suggestion
  # content_length_limit = "0KB"
```
to indicate that we support size suffices here.

---

I have made the same correction in this commit.
https://github.com/influxdata/telegraf/commit/89c595cc3636435fa25da2fb935badd2c13129a8